scripts/oslextractmeta.py

```diff
@@ -337,16 +345,16 @@ def cli():
     parser = argparse.ArgumentParser(
         formatter_class=argparse.RawDescriptionHelpFormatter,
         description='''
-        oslextractmetadata stores the user interface and metadata of a 
-        compiled OSL (openshadinglanguage) shader(s) into a JSON file.
-        The JSON dictionary consists of a 'header' and a 'shader' part.
-        jsondict['shader'] will return a dictionary with all shaders. The
-        user interface of the shader is stored as a sub-dictionary, the
-        metadata can be retrieved using the 'ui' key on the elements, e.g.:
-
-        for x in jsondict['shaders'].values():
-            print x['ui']
-        ''')
+oslextractmetadata stores the user interface and metadata of a
+compiled OSL (openshadinglanguage) shader(s) into a JSON file.
```

Comment:
You will end up with `of acompiled OSL` here

Reply:
Wouldn't the newline be preserved in a triple-quoted string?

Reply:
Good point. I honestly don't know ^^

Reply:
Just confirmed, the newline is printed in the help text.